fortnite_bot.py

```diff
@@ -132,6 +133,45 @@ async def rate(ctx, silent=False):
     """ Rate how good opponents are today """
     await stats.rate_opponent_stats_today(ctx)
 
+@bot.command(name=commands.UPGRADE_COMMAND, help=commands.UPGRADE_DESCRIPTION,
+             aliases=commands.UPGRADE_ALIASES)
+async def upgrade(ctx, silent=False):
+    """ Show map of upgrade locations """
+    await ctx.send("https://img.fortniteintel.com/wp-content/uploads/2021/03/17151906/Fortnite-Season-6-upgrade-locations.jpg.webp")
```

Comment:
I'll/we can move these to `interactions.py` in the future I think, so the "logic" will be in the client files and this will just be a driver